Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
+{
+    $PSScriptRoot = [System.IO.Path]::GetDirectoryName($MyInvocation.MyCommand.Path)
+}
+
+$ErrorActionPreference = 'Stop'
```

Comment:
This line might not be a good idea because changing the ErrorActionPref will cause the module to behave differently in testing than when being used in a real DSC environment.